### PR TITLE
Added support for getting a single private link resource

### PR DESCRIPTION
**Pull request by BethanyZhou**
## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [x] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/Network/Network/help/Get-AzPrivateLinkResource.md [hunk 1]
```diff
@@ -75,7 +90,7 @@ The private link resource type.
 Type: System.String
 Parameter Sets: ByResource
 Aliases:
-Accepted values:
+Accepted values: Microsoft.AppConfiguration/configurationStores, Microsoft.Automation/automationAccounts, Microsoft.Batch/batchAccounts, Microsoft.Cache/redisEnterprise, Microsoft.CognitiveServices/accounts, Microsoft.Compute/diskAccesses, Microsoft.ContainerRegistry/registries, Microsoft.DBforMariaDB/servers, Microsoft.DBforMySQL/servers, Microsoft.DBforPostgreSQL/servers, Microsoft.Devices/IotHubs, Microsoft.DocumentDB/databaseAccounts, Microsoft.EventGrid/topics, Microsoft.EventGrid/domains, Microsoft.EventHub/namespaces, Microsoft.HealthcareApis/services, Microsoft.Insights/privateLinkScopes, Microsoft.KeyVault/vaults, Microsoft.Media/mediaservices, Microsoft.Migrate/assessmentProjects, Microsoft.Migrate/migrateProjects, Microsoft.Network/applicationgateways, Microsoft.OffAzure/masterSites, Microsoft.Purview/accounts, Microsoft.Search/searchServices, Microsoft.ServiceBus/namespaces, Microsoft.SignalRService/signalr, Microsoft.Sql/servers, Microsoft.Storage/storageAccounts, Microsoft.StorageSync/storageSyncServices, Microsoft.Synapse/workspaces, Microsoft.Web/sites, Microsoft.Web/hostingEnvironments
```
Comment (dingmeng-xue) [inline src/Network/Network/help/Get-AzPrivateLinkResource.md:93]: This part should not be added. Please review.

**Review by dingmeng-xue — CHANGES_REQUESTED** [review]

File: src/Network/Network/help/Get-AzPrivateLinkResource.md (hunk 1 above)
Comment (BethanyZhou) [inline src/Network/Network/help/Get-AzPrivateLinkResource.md:93]:
```suggestion
Accepted values: 
```

File: src/Network/Network/help/Get-AzPrivateLinkResource.md [hunk 2]
```diff
@@ -75,7 +90,7 @@ The private link resource type.
 Type: System.String
 Parameter Sets: ByResource
 Aliases:
-Accepted values:
+Accepted values:
```
Comment (BethanyZhou) [inline src/Network/Network/help/Get-AzPrivateLinkResource.md:93]:
```suggestion
Accepted values:
```

File: src/Network/Network/help/Get-AzPrivateLinkResource.md (hunk 1 above)
Comment (BethanyZhou) [reply src/Network/Network/help/Get-AzPrivateLinkResource.md]: Sure, have no idea where these accepted values from. Removed.

**Review by dingmeng-xue — APPROVED** [review]

**Comment by BethanyZhou** [comment]
Hi @VeryEarly , it seems like the failed test doesn't matter with this pr. Could you take a look?

**Pull request merged**